README.md

```diff
@@ -1974,6 +1974,13 @@ $ brew unlink trivy && brew uninstall trivy
 $ brew install aquasecurity/trivy/trivy
 ```
 
+### Error: x509: certificate signed by unknown authority
+`TRIVY_INSECURE` can be used to allow insecure server connections when using SSL.
```

Comment:
IIRC it's for insecure registry server. Not for Trivy server in client-server right? Maybe we could mention that explicitly.

Reply:
Thanks. Updated.